iterators.tex

```diff
@@ -623,31 +650,186 @@
 \textit{p}).
 \exitexample
 
-\indexlibrary{\idxcode{Iterator}}%
-\begin{codeblock}
+{\color{newclr}
+\rSec2[iterators.sizediterator]{Concept SizedIterator}
+\pnum
+The \tcode{SizedIterator} concept refines \tcode{Iterator} with the
+capability to determine the distance between two iterators that denote
+elements of the same range in constant time with the \tcode{-} operator.
+
+\indexlibrary{\idxcode{SizedIterator}}%
+\begin{itemdecl}
+  template <class>
+  constexpr bool disable_sized = false;
```

Comment:
I think I can buy the ADL argument, although in that case one could `delete` a more-specialized `op-` overload as a countermeasure. I don't buy the symmetry argument as much; it should take two types to opt in and enable a behavior, but any one is enough to opt out. In any case I don't see this change as a huge detriment. I'll merge two concepts into one and split one trait into two.